Gemfile

```diff
@@ -2,8 +2,16 @@ source 'https://rubygems.org'
 
 ruby '2.2.1'
 
+gem 'data_mapper'
+gem 'dm-postgres-adapter'
+gem 'rack-flash3'
+gem 'sinatra'
+
 group :test do
+  gem 'byebug'
+  gem 'database_cleaner'
   gem 'rspec'
+  gem 'capybara'
   gem 'cucumber'
```

Comment:
You've included cucumber and all the Cucumber setup files but all of your feature tests are in RSpec.  Don't include unnecessary files in your solution